### PR TITLE
Add govuk-frontend-jinja and digitalmarketplace-govuk-frontend

### PR DESCRIPTION
**Pull request by benvand**
Screen Shot 2019-10-16 at 09 18 23

Following steps on ticket:
https://trello.com/c/ZtEH5XcG/20-2-add-govuk-frontend-jinja-and-digitalmarketplace-govuk-frontend-to-brief-responses-frontend

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -46,13 +47,13 @@ pycparser==2.19
...
-pytz==2019.2
+pytz==2019.3
 PyYAML==3.13
 requests==2.22.0
 s3transfer==0.2.1
 six==1.12.0
 unicodecsv==0.14.1
 urllib3==1.25.6
-Werkzeug==0.15.6
+Werkzeug==0.16.0
```
Comment (katstevens) [inline requirements.txt]: We decided to do this, but can we do it in a different PR/release in case we have to roll back?

File: gulpfile.js [hunk 2]
```diff
@@ -251,6 +253,15 @@ gulp.task('test', function () {
...
 })
 
+gulp.task(
+  'copy:govuk_frontend_assets:fonts',
+  copyFactory(
+    'fonts from the GOVUK frontend assets',
+    govukFrontendFontsFolder,
+    staticFolder + '/fonts'
+  )
+)
```
Comment (lfdebrux) [inline gulpfile.js]: We also need a task to copy the images, (see https://github.com/alphagov/digitalmarketplace-admin-frontend/commit/b577b4d60a890d910984312c2f20d34184931397#diff-b9e12334e9eafd8341a6107dd98510c9R254).

I'll update the ticket to make it clearer.

File: requirements.txt (hunk 1 above)
Comment (benvand) [reply requirements.txt]: https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/pull/155

File: gulpfile.js (hunk 2 above)
Comment (lfdebrux) [reply gulpfile.js]: You can merge the PR without the task though.

File: requirements.txt (hunk 1 above)
Comment (benvand) [reply requirements.txt]: Nice, rebase worked

File: requirements.txt [hunk 3]
```diff
@@ -12,6 +12,7 @@ gds-metrics==0.2.0
 git+https://github.com/alphagov/digitalmarketplace-utils.git@48.5.0#egg=digitalmarketplace-utils==48.5.0
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@6.0.0#egg=digitalmarketplace-content-loader==6.0.0
 git+https://github.com/alphagov/digitalmarketplace-apiclient.git@21.4.0#egg=digitalmarketplace-apiclient==21.4.0
+git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.3.0-alpha#egg=govuk-frontend-jinja==0.3.0-alpha
```
Comment (aliuk2012) [inline requirements.txt]: Sorry about this it was our fault. This should be govuk-frontend-jinja 0.4.0-aplha (https://github.com/alphagov/govuk-frontend-jinja/releases) We will be updating admin-frontend

**Comment by aliuk2012** [comment]
# 🚀 Lets do it
